pyproject.toml

```diff
@@ -31,3 +39,4 @@ markers = [
 
 [tool.black]
 line-length = 79
+target-version = ["py38"]
```

Comment:
this isn't required, black will infer it from the `requires-python` value

see https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#t-target-version